protokube/pkg/gossip/mesh/gossip.go

```diff
@@ -41,14 +41,16 @@ type MeshGossiper struct {
 
 func NewMeshGossiper(listen string, channelName string, nodeName string, password []byte, seeds gossip.SeedProvider) (*MeshGossiper, error) {
 
-	connLimit := 64
+	connLimit := 0 // 0 means no limit
 	gossipDnsConnLimit := os.Getenv("GOSSIP_DNS_CONN_LIMIT")
```

Comment:
Any reason to still have the ENV var at all? Or are we expecting to have user limit the number of peers each node has in case they run into issues and the gossip pool of each node becomes too large?